src/app/CommandInputField.tsx

```diff
@@ -85,6 +93,16 @@ export const CommandInputField = observer((props: { controller: BottomPanelContr
       const selected = controller.selected;
       const autoCompletionValues = controller.autoCompletionValues;
 
+      if (selected === null && autoCompletionValues.length <= 0 && lastCmd.length > 0) {
+        if (lastCmdIndex.get() + 1 < lastCmd.length) {
```

Comment:
Refactor it such that the get and set method use once only

Reply:
Updated, thank you